pom.xml

```diff
@@ -156,6 +157,20 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>${maven-source-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <id>attach-sources</id>
+                            <phase>verify</phase>
```

Comment:
just for my knowledge : why verify ? it seems to be attached to the "package" phase by default

Reply:
If I quote the plugin documentation : 
`We are using the verify phase here because it is the phase that comes before the install phase, thus making sure that the sources jar has been created before the install takes place.`

https://maven.apache.org/plugins/maven-source-plugin/usage.html